homeassistant/helpers/area_registry.py

```diff
@@ -154,24 +154,23 @@ def _data_to_save(self) -> Dict[str, List[Dict[str, Optional[str]]]]:
         return data
 
 
-@bind_hass
-async def async_get_registry(hass: HomeAssistantType) -> AreaRegistry:
-    """Return area registry instance."""
-    reg_or_evt = hass.data.get(DATA_REGISTRY)
+@callback
+def async_get(hass: HomeAssistantType) -> AreaRegistry:
+    """Get area registry."""
+    return cast(AreaRegistry, hass.data[DATA_REGISTRY])
 
-    if not reg_or_evt:
-        evt = hass.data[DATA_REGISTRY] = Event()
 
-        reg = AreaRegistry(hass)
-        await reg.async_load()
+async def async_load(hass: HomeAssistantType) -> None:
+    """Load area registry."""
+    assert DATA_REGISTRY not in hass.data
```

Comment:
Is this assert for developers to not make mistakes? Assert can be turned off in production code.

Reply:
Yes, it's only intended for developers / tests.
In production, the registries are loaded once by bootstrap.py.